nix/lib.nix

```diff
@@ -21,12 +21,10 @@
             default = "./data/${name}";
             description = "The directory where all data for `${service}.<name>` is stored";
           };
-          namespace = lib.mkOption {
-            description = ''
-              Namespace for the ${service} service
-            '';
-            default = "${service}.${name}";
-            type = lib.types.str;
+          processSettings = lib.mkOption {
+            type = lib.types.lazyAttrsOf lib.types.deferredModule;
```

Comment:
lazyAttrsOf because a service can define multiple processes (postgres, postgres-init for example)